l10n_it_rea/models/l10n_it_rea_mixin.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2020 Tecnativa - Víctor Martínez
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+from odoo import models, fields
+
+
+class L10nItReaMixin(models.AbstractModel):
+    _name = 'l10n_it_rea.mixin'
```

Comment:
https://github.com/OCA/l10n-italy/pull/2017#pullrequestreview-561539307

Reply:
Because this addon haven't `l10n_it_account` dependency.

Reply:
@eLBati is it OK to add such dependency on the modules instead?

Reply:
https://github.com/OCA/l10n-italy/pull/2017#discussion_r551947927

I think it's better  not to add `l10n_it_account` as a dependency of modules like  `l10n_it_rea`, because some database could use these basic modules without using accounting.

Reply:
OK, I see, then maybe a `l10n_it_base` containing this mixin can be an option, or to duplicate code.